src/Services/HttpRequestService.php

```diff
@@ -157,6 +157,9 @@ protected function sendRequest($method, $url, array $data = [], array $headers =
         return $response;
     }
 
+    /**
+     * @codeCoverageIgnore
+     */
```

Comment:
could we mock logger instead?

Reply:
I added deprecation and text that better to use additional packages for logging